spec/head_spec.js

```diff
@@ -1,10 +1,26 @@
-var head = require("../src/head.js")
+import { head } from '../src/head'
 
 describe('head', ()=> {
   it('returns the first element in an array', () => {
-    expect(head([1,2,3,4,5,])).toEqual(1)
-    expect(head([2,3,4,5,])).toEqual(2)
-    expect(head([3,4,5,])).toEqual(3)
+    const array = [1,2,3,4,5,]
```

Comment:
I would omit the separate variable declarations here since they're only used once.